src/protagonist/DLCS.Repository/DlcsContext.cs

```diff
@@ -630,6 +634,55 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
             }
         );
         
+        modelBuilder.Entity<DeliveryChannelPolicy>(entity =>
+        {
+            entity.Property(e => e.Name).IsRequired();
+            entity.Property(e => e.Customer).IsRequired();
+            entity.Property(e => e.System).IsRequired();
+
+            entity.HasIndex(e => new { e.Customer, e.Name, e.Channel }).IsUnique();
+
+            entity.Property(e => e.Name).HasMaxLength(500);
+            
+            entity.Property(e => e.Modified).HasColumnType("timestamp with time zone")
+                .IsRequired();
+
+            entity.Property(e => e.Modified).HasColumnType("timestamp with time zone")
+                .IsRequired();
+
+            entity.Property(e => e.Channel)
+                .IsRequired()
+                .HasMaxLength(100);
+        });
+        
+        modelBuilder.Entity<ImageDeliveryChannel>(entity =>
+        {
+            entity.HasKey(e => e.Id);

```

Comment:
This should be redundant and done by convention (https://learn.microsoft.com/en-us/ef/core/modeling/keys?tabs=data-annotations#configuring-a-primary-key)